rotkehlchen/tests/api/test_makerdao_vaults.py

```diff
@@ -891,3 +894,48 @@ def test_query_vaults_usdc_strange(rotkehlchen_api_server, ethereum_accounts):
     details = assert_proper_response_with_result(response)
     expected_details = [vault_7538_details]
     assert_serialized_lists_equal(expected_details, details)
+
+
+@pytest.mark.vcr()
+@pytest.mark.parametrize('ethereum_modules', [['makerdao_vaults']])
+@pytest.mark.parametrize('mocked_proxies', [{
```

Comment:
the prices are getting mocked right? Could you explicitly state it?

Reply:
Prices don't matter in this test, so I think there is no need to mention them

Reply:
is not to mention then but set to True the variable that mocks them always to make sure that no extra external query is made

Reply:
Prices are mocked by default https://github.com/rotki/rotki/blob/cc0db0161e7f3534d5673b0d5b9b1c3be5c6f348/rotkehlchen/tests/fixtures/accounting.py#L66

Reply:
But I will also explicitly specify it here